packages/api-v4/src/cloudpulse/types.ts

```diff
@@ -132,3 +132,8 @@ export interface ServiceTypes {
 export interface ServiceTypesList {
   data: ServiceTypes[];
 }
+
+export interface DataSet {
+  [label: string]: number;
```

Comment:
Is this needed, how they are handling it for linode summary ? , they should have also come across such types right?

Reply:
https://github.com/nikhagra-akamai/manager/blob/develop/packages/manager/src/components/AreaChart/types.ts

they have hardcoded everything